App.js

```diff
@@ -1,25 +1,29 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import SvgComponent from "./LogoSvg";
+import LogoSvg from "./LogoSvg";
+import Quotes from "./Quotes";
+import AnimateSvg from './AnimateSvg';
+import AnimateQuotes from './AnimateQuotes';
 
 export default function App() {
-
   return (
-    <View style={styles.container}>
-      <SvgComponent />
+    <View style={styles.window}>
+      <AnimateSvg>
+        <LogoSvg />
+      </AnimateSvg>
+      <AnimateQuotes>
+        <Quotes />
+      </AnimateQuotes>
```

Comment:
Added wrapper components and relevant children serving as the framework of the app.